python/llama-index-server/llama_index/server/api/utils/workflow.py

```
@@ -0,0 +1,115 @@
+    JsonSerializer,
+    Workflow,
+)
+from llama_index.core.workflow.handler import WorkflowHandler
+from llama_index.server.models.hitl import HumanInputEvent
+from llama_index.server.utils.class_meta_serialization import (
+    type_from_identifier,
+    type_identifier,
+)
+
+
+class HITLWorkflowService:
```

Pylint (3.3.7)

[convention] Missing class docstring

(C0115)